docs/testplan/server-side-copy-testplan.md

```diff
@@ -0,0 +1,53 @@
+|                        | Object Deleted After Copy- inline segment                     | If a user uploads an inline segment, server side copies it from the source object to the destination object and then deletes the original source object, then an error is not shown                                                                                     |          |
+|                        | Bandwidth & Storage- remote segment                           | If a user uploads an remote segment and then server side copies from the source object to the destination object, then bandwidth and storage should stay the same before the operation                                                                                  |          |
+|                        | File Size- remote segment                                     | If a user uploads an remote segment and then server side copies from the source object to the destination object, then the destination object should be the same size as the original source object                                                                     |          |
+|                        | Object Deleted During Copy- remote segment                    | If a user uploads an remote segment and then server side copies from the source object to the destination object all while deleting the original source object, then an error is thrown                                                                                 |          |
+|                        | Object Deleted After Copy- remote segment                     | If a user uploads an remote segment, server side copies it from the source object to the destination object and then deletes the original source object, then an error is not shown                                                                                     |          |
+|                        | Copy 10000 Or More Parts                                   | If a user uploads an object with 10000 parts or more and server side copies it from the source object to the destination object, it should be possible                                                                                                               |          |
+|                        | Copy 10000 Or More Segments                                   | If a user uploads an object with 10000 segments or more and server side copies it from the source object to the destination object, it should be possible                                                                                |          |
+|                        | Copy Objects More than 5GB                                    | If objects are more than 5GB in size, then upload-part-copy api must be used for server side copy                                                                                                                                                       |          |
```

Comment:
From the satellite perspective, I don't think we will have a limit on size of the object, the limit will only on segment count.

Reply:
Updated